pkgs/development/libraries/libxklavier/default.nix

```
@@ -13,14 +13,12 @@ stdenv.mkDerivation rec {
 
   patches = [ ./honor-XKB_CONFIG_ROOT.patch ];
 
-  outputs = [ "out" "dev" "devdoc" ];
+  outputs = [ "out" "dev" ] ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [ "devdoc" ];
```

nit, I think it would be a bit more readable to do something like

```suggestion
  outputs = [ "out" "dev" ] ++ lib.optionals withDoc [ "devdoc" ];
```
where `withDoc ? stdenv.buildPlatform == stdenv.hostPlatform`

---

Done. Thanks for the suggestion!